src/Symfony/Bundle/FrameworkBundle/Command/ContainerLintCommand.php

```diff
@@ -69,6 +90,10 @@ private function getContainerBuilder(): ContainerBuilder
         $kernel = $this->getApplication()->getKernel();
 
         if (!$kernel->isDebug() || !(new ConfigCache($kernel->getContainer()->getParameter('debug.container.dump'), true))->isFresh()) {
+            if (!$kernel instanceof Kernel) {
+                throw new RuntimeException('This command does not support the application kernel.');
```

Comment:
`This command does not support the application kernel: "%s" does not extend "%s".`
(same in similar messages)

Reply:
Done on 4.4, see https://github.com/symfony/symfony/pull/34985. I'm gonna wait for a merge on master to finish this one.